CONTRIBUTING.md

```diff
@@ -289,5 +289,8 @@ upon publishment.*
 
 ## Release schedule
 
-This project follows a two-week release cycle. Note that a release might be skipped if there's no
-significant changes to release.
+This project follows a weekly release schedule, with planned releases every Tuesday. If there is
+no significant content to ship on a given release day, we automatically defer to the next scheduled
+window. Should we encounter any CI issues on the day of a release, we postpone until after those
```

Comment:
s/, we postpone until.*\./, we either release immediately after the issue is resolved (if a release is urgent) or skip the release (if no urgent changes were planned)./

Reply:
How do you define an "urgent release" when it comes to regular releases and more specifically regarding features/improvements, i.e. not bug/security fixes? I mean I consider any feature/improvement (apart from experimental stuff naturally) equally important when it comes to releasing it, no skipping lines.

How about:
_we either release immediately after the issue is resolved or skip the release based on how long it takes to get the CI green again_ ?

Reply:
"Urgent" is a blocking or security bug, everything else is regular. Urgent can coincide with regular release. "Skip ... based on how long it takes to..." shifts focus from release nature to CI block nature which I think is not right. I would have rather kept the original wording than update it this way: I tried to make the sentence a bit easier to digest, but it is fine as it is now.

Reply:
> Urgent can coincide with regular release.

True. I will update with your suggestion, but will swap the last 2 sentences as it makes more sense that way with the suggestion.